.npmignore

```diff
@@ -1,19 +1,35 @@
-#NPM
+# NPM
 node_modules
 npm-debug.log
 
-#Testing
+# Testing
 test
 coverage
+.nyc_output
 
-#CI
+# CI
 .travis.yml
 .codeclimate.yml
+.appveyor.yml
 
-#DevOps
+# DevOps
 scripts
 
 # TypeScript
 *.js.map
 *.ts
 !*.d.ts
+tsconfig.json
+tslint*.json
```

Comment:
@jamesrichford I think it may be better to explicitly ignore `tslint.json` and `tslint.test.json` just to be clear. We've got a set number of files matching the glob, rather than when we ignore `*.js` which isn't a set number of files.